servicepulse/index.md

```diff
@@ -25,6 +25,8 @@ The Dashboard provides a visual overview of the current state of the monitored N
 
 ServicePulse automatically detects what endpoints exist in the system by analyzing metadata from [audited messages](/nservicebus/operations/auditing.md). All detected endpoints will, by default, be listed in the Endpoints tab, but will not be automatically monitored.
 
+NOTE: Endpoints will not appear unless messages from the endpoint have been successfully processed by ServiceControl.
```

Comment:
soon we'll be also sending metrics as messages, so I don't think it's necessary to mention heartbeats explicitly